bundles/io/org.openhab.io.gcal/src/main/java/org/openhab/io/gcal/internal/GCalEventDownloader.java

```diff
@@ -59,12 +81,14 @@
 public class GCalEventDownloader extends AbstractActiveService implements ManagedService {
 
     private static final String GCAL_SCHEDULER_GROUP = "gcal";
+    private static final String TOKEN_PATH = "gcal";
+    private static final String TOKEN_STORE_USER_ID = "openhab";
 
     private static final Logger logger = LoggerFactory.getLogger(GCalEventDownloader.class);
 
-    private static String username = "";
-    private static String password = "";
-    private static String url = "";
+    private static String client_id = "923938667558-3agtjnqnlfq4ku2v6veoradnt88291du.apps.googleusercontent.com";
+    private static String client_secret = "ow_sZCtGTD316vnWm7zOGXfL";
+    private static String calendar_name = "primary";
```

Comment:
OK, but I worry that this creates a dependency on your credentials which might stop working someday.  I think it might make sense to either use credentials controlled by the openHAB Foundation as defaults, or simply not provide defaults.  (A similar situation recently cropped up with Twitter integration, and it's still unresolved due to unforeseen constraints.)  @kaikreuzer, what is your opinion?